src/App.js

```diff
@@ -98,25 +99,25 @@ function App() {
+        { exact: true, path: "/", render: () => (<HomePage contentRef={contentRef} homePageProps={homeProps} classname="main" />) },
+        { path: "/about-standard", render: () => <GenericContentPage cmsLocation={ABOUT_PAGE} articleType="about" contentRef={contentRef} /> },
+        { path: "/how-it-works/:slugField", render: ({ match }) => <GenericContentPage cmsLocation={`/pages?slugfield=${match.params.slugField}`} articleType="page" parent={{ path: `/how-it-works`, title: `How it works` }} contentRef={contentRef} /> },
+        { path: "/how-it-works", render: () => <GenericLandingPage cmsLocation={process.env.REACT_APP_HOW_WORKS} articleType="HowItWorks" contentRef={contentRef} /> },
+        { path: "/community/case-studies/:slugField", render: ({ match }) => <CaseStudyPage slugField={match.params.slugField} contentRef={contentRef} /> },
+        { path: "/community/case-studies", render: () => <LandingPage styleName="main" parent={{ path: `/community`, title: `Community` }} contentRef={contentRef} /> },
+        { path: "/community/standard-community", render: () => <WhoIsUsing styleName="main" parent={{ path: `/community`, title: `Community` }} contentRef={contentRef} /> },
+        { path: "/community", render: () => <GenericLandingPage cmsLocation={process.env.REACT_APP_COMMUNITY_PAGE} articleType="communityPage" contentRef={contentRef} /> },
+        { path: "/contact-us", render: () => <GenericContentPage cmsLocation={CONTACT_PAGE} articleType="contactUs" contentRef={contentRef} /> },
+        { path: "/accessibility-statement", render: ({ location }) => <GenericContentPage cmsLocation={`/pages?slugfield=${location.pathname.substring(1)}`} articleType="page" contentRef={contentRef} /> },
+        { path: "/privacy-policy", render: ({ location }) => <GenericContentPage cmsLocation={`/pages?slugfield=${location.pathname.substring(1)}`} articleType="page" contentRef={contentRef} /> },
+        { path: "/terms-conditions", render: ({ location }) => <GenericContentPage cmsLocation={`/pages?slugfield=${location.pathname.substring(1)}`} articleType="page" contentRef={contentRef} /> },
+        { path: "/show-error", render: () => <GenericErrorPage contentRef={contentRef} /> },
+        { path: "/open-referral-uk-video-transcript", render: ({ location }) => <GenericContentPage cmsLocation={`/pages?slugfield=${location.pathname.substring(1)}`} articleType="page" contentRef={contentRef} /> },
+        { path: "/dashboard", render: () => <Dashboard contentRef={contentRef} /> },
+        { path: "/devdashboard", render: () => <Dashboard overrideData={mockData} contentRef={contentRef} /> },
+        { path: "/register", render: ({ location }) => <RegisterPage cmsLocation={`/pages?slugfield=${location.pathname.substring(1)}`} articleType="page" contentRef={contentRef} /> },
+        { path: "/preview", render: () => <Preview contentRef={contentRef} /> },
+        { path: "/404", render: () => <NotFound contentRef={contentRef} /> }
     ];
```

Comment:
I note r.render is expected on lines 167?  Are we good?

Reply:
Removed the redirect for standard-community on line 167 since there is another path being used to the same content /community/standard-community and there is no reference of it being used anywhere else.

Reply:
This is the line content I was referring to on 167
{routes.map(r => <Route key={r.path} exact={r.exact} path={r.path} component={r.component} render={r.render} />)}
so the component prop is that still been referenced anywhere?  
It's not a show stopper. but at a later date can be removed if we are sure it is not being used.  

But otherwise I have tested and all looks good.